src/components/BottomMenuModal/SendModal/SendModalTokensTabView.tsx

```
@@ -23,10 +11,21 @@ import {
   ClipboardTick as IconClipboardTick,
   Flash as IconFlash,
 } from 'iconsax-react';
-import React, { useContext, useEffect } from 'react';
+import { isNaN } from 'lodash';
```

Biome (2.1.2)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)